winui/Masked-TextBox/Culture.md

```diff
@@ -1,10 +1,76 @@
 ---
 layout: post
-title: Localization in WinUI MaskedTextBox control | Syncfusion
-description: Learn all about the localization in the MaskedTextBox (SfMaskedTextBox) control for the specific set of symbols such as '.', ',', '/', ':' and '$'.
+title: Culture in WinUI MaskedTextBox control | Syncfusion
+description: Learn all about the culture in the MaskedTextBox (SfMaskedTextBox) control for the specific set of symbols such as '.', ',', '/', ':' and '$'.
 platform: WinUI
```

Comment:
@Vignesh-SF3580 Learn all about how to set the different culture support for currency symbols, date separators, and decimal separators in the MaskedTextBox (SfMaskedTextBox) control.

Reply:
Changed the description like this.